include/substrait-mlir/Dialect/Substrait/IR/CMakeLists.txt

```diff
@@ -7,8 +7,16 @@ mlir_tablegen(SubstraitOpInterfaces.cpp.inc -gen-op-interface-defs)
 mlir_tablegen(SubstraitTypeInterfaces.h.inc -gen-type-interface-decls)
 mlir_tablegen(SubstraitTypeInterfaces.cpp.inc -gen-type-interface-defs)
 add_public_tablegen_target(MLIRSubstraitInterfacesIncGen)
-add_dependencies(MLIRSubstraitDialect MLIRSubstraitInterfacesIncGen)
```

Comment:
Kudos for making this work! I hadn't had this on the radar.

On detail, though: the names of the generated files don't follow the conventions yet.
And an even more minor nit: Let's move this up for alphabetical order and add an independent `add_depencencies`:

```cmake
set(LLVM_TARGET_DEFINITIONS SubstraitOps.td)
mlir_tablegen(SubstraitEnums.h.inc -gen-enum-decls)
mlir_tablegen(SubstraitEnums.cpp.inc -gen-enum-defs)
add_public_tablegen_target(MLIRSubstraitEnumsIncGen)
add_dependencies(MLIRSubstraitDialect MLIRSubstraitEnumsIncGen)
```